article_store/core/urls.py

```diff
@@ -0,0 +1,15 @@
+from django.conf import settings
+from django.conf.urls.static import static
+from django.contrib import admin
+from django.urls import path, include
+
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+    path('articles/', include('articles.urls')),
```

Comment:
This might need to be part of deployment, so you could have `/blog-articles` or whatever instead (related to https://github.com/libero/walking-skeleton/pull/22/files#r188602069).